content/posts/2022/reana-0.8.1.md

```diff
@@ -0,0 +1,204 @@
+---
+title: "REANA 0.8.1 is released"
+date: 2022-02-15T09:00:00+01:00
+tags:
+  - release
+  - client
+  - cluster
+---
+
+We are glad to announce the release of REANA 0.8.1, a minor update that allows
+users to set job timeout limits, improves workspace web page file name
+filtering, allows displaying workspace HTML reports directly, and more!
+
+<!--more-->
+
+## What's new for the users?
+
+### Configurable job timeout limits
+
+If your workflow launches many hundreds of jobs and some of these may be
+"running away" --- for example a job enters a state of infinite loop waiting
```

Comment:
```suggestion
"running away" --- for example, a job enters a state of infinite loop waiting
```

Reply:
I would like to avoid using comma inside triple-dashes, so changed to "for example when a job enters..."